test/jdk/java/util/Currency/CurrencyTest.java

```diff
@@ -84,23 +84,34 @@ private static Stream<String> validCurrencies() {
         public void invalidCurrencyTest(String currencyCode) {
             IllegalArgumentException ex = assertThrows(IllegalArgumentException.class, () ->
                     Currency.getInstance(currencyCode), "getInstance() did not throw IAE");
-            assertEquals("The input currency code is not a" +
-                    " valid ISO 4217 code", ex.getMessage());
+            assertEquals("The input currency code: \"%s\" is not a valid ISO 4217 code"
+                    .formatted(currencyCode), ex.getMessage());
         }
 
         private static Stream<String> non4217Currencies() {
             return Stream.of("AQD", "US$");
         }
 
+        // Provide 3 length code, but first 2 chars should not be able to index
+        // the main table
+        @Test
+        void invalidCountryInCodeTest() {
+            var badCode = "..A";
+            IllegalArgumentException ex = assertThrows(IllegalArgumentException.class, () ->
+                    Currency.getInstance(badCode), "getInstance() did not throw IAE");
+            assertEquals("The country code: \"%s\" is not a valid ISO 3166 code"
```

Comment:
Since the test is not parameterized, we can simply use ".." inside the expected string literal.